packages/ses/test/hermes-smoke.js

```diff
@@ -0,0 +1,44 @@
+// TODO: equivalent of:
+// import '../dist/ses.cjs';
+// import '../src/assert-shim.js';
+// Since Hermes has no native support for I/O
+
+// Test lockdown
+
+lockdown();
+
+// Test Compartment
+
+const c = new Compartment();
+
+c.evaluate('1+1');
+c.evaluate("const c2 = new Compartment(); c2.evaluate('1+2')");
+
+// Test importHook and resolveHook
+
+// https://github.com/facebook/hermes/blob/main/doc/Features.md
+// In Progress: ES modules (`import` and `export`)
+
+const resolveHook = '';
```

Comment:
```suggestion
const resolveHook = (a) => a;
```

Reply:
done in https://github.com/leotm/endo/pull/1/commits/4fa7963e4434169a66699b1496984856b7f115db